apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -95,6 +95,9 @@ type OpenShiftFeatureGates struct {
 
 	// OauthProxy define options for the oauth proxy feature.
 	OauthProxy OauthProxyFeatureGates `json:"oAuthProxy,omitempty"`
+
+	// DistributorServingCertsService
```

Comment:
A more explanation would be helpful? 

What does the config enable? Does it inject OCP service CA to distributor pod? How is it handled for the gateway?